nhost/compose/config.go

```diff
@@ -315,6 +315,8 @@ func (c Config) functionsServiceEnvs() env {
 	e := env{}
 	e.merge(env{
 		"NHOST_BACKEND_URL":    c.envValueNhostBackendUrl(),
+		"NHOST_SUBDOMAIN":      "localhost",
+		"NHOST_REGION":         "",
```

Comment:
The point is to be explicit about what vars are available and to have a 1-1 mapping with production. The value is irrelevant IMO.

```js
const nhost = new NhostClient({
  subdomain: process.env.NHOST_SUBDOMAIN,
  region: process.env.NHOST_REGION
})
```

This is what users will be doing most of the time, and I just noticed that `process.env.NHOST_REGION` already returns `null` in case it is not explicitly set, maybe we should change it to null instead of the empty string.

Reply:
> maybe we should change it to null instead of the empty string

it's go, not javascript 😄